packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -84,17 +84,17 @@ class CameraValue {
   ///
   /// This is null while the controller is not in an error state.
   /// When [hasError] is true this contains the error description.
-  final String errorDescription;
+  final String? errorDescription;
 
   /// The size of the preview in pixels.
   ///
   /// Is `null` until  [isInitialized] is `true`.
```

Comment:
Maybe we can update typos all together?

```suggestion
  /// Is `null` until [isInitialized] is `true`.
```